### ISSUE TITLE
Add subtasks to the enlisted tasks.

### ISSUE THREAD

**Issue by tusharnankani** [tusharnankani/ToDoList#6]
SubTasks: 
- Adding subtasks to the enlisted tasks, could be an option.
- The layout isn't thought out yet.

Current thoughts for a layout:
- Either a popup when clicked on the To-Do, and further description can be added.
- Or an accordion, something like this.
![Screenshot 2020-11-10 205550](https://user-images.githubusercontent.com/61280281/98694179-2a33be00-2397-11eb-86a9-207bd4d0b3d5.png)

**Comment by ratatatCat** [comment]
I'd be interested in helping out with this one. :-)

**Comment by tusharnankani** [comment]
> I'd be interested in helping out with this one. :-)

Great. Have you thought of a layout yet?

**Comment by ratatatCat** [comment]
I was thinking of going more with the accordion proposal -> I'm an avid list maker and definitely prefer the dropdown approach more than the popup when using to-do lists in general. What do you think?

**Comment by tusharnankani** [comment]
> I was thinking of going more with the accordion proposal -> I'm an avid list maker and definitely prefer the dropdown approach more than the popup when using to-do lists in general. What do you think?

Alright! Go ahead!

**Comment by tusharnankani** [comment]
Hey @ratatatCat 
Any updates?

**Comment by ratatatCat** [comment]
Ah! Sorry, I ended up getting pretty busy with work and got distracted. If it's not too late, I am off next week and could finish it up.

**Comment by tusharnankani** [comment]
Yeah, no issues. Was just taking updates since it has been quite some time. 

> If it's not too late, I am off next week and could finish it up.

Yes, go ahead!

**Comment by ratatatCat** [comment]
Awesome! I'm almost finished and will have a PR ready tomorrow or Tuesday. Thanks!
